include/seqan3/search/fm_index/bi_fm_index_cursor.hpp

```diff
@@ -497,12 +486,10 @@ class bi_fm_index_cursor
      * No-throw guarantee.
      */
     template <std::ranges::RandomAccessRange seq_t>
```

Comment:
BidirectionalRange

Reply:
?

Reply:
I think even `InputRange` is enough since we only access the seq once via the for loop
```
for (auto it = first; it != last; ++it)
```